client/js/upload.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const $ = require("jquery");
+const socket = require("./socket");
+const SocketIOFileUpload = require("socketio-file-upload/client");
+const instance = new SocketIOFileUpload(socket);
+
+instance.listenOnInput(document.getElementById("upload-input"));
+instance.listenOnDrop(document.body);
+
+$("#upload").on("click", () => {
+	$("#upload-input").trigger("click");
+});
+
+instance.addEventListener("complete", () => {
+	// Reset progressbar
+	$("#upload-progressbar").width(0);
+});
+
+instance.addEventListener("progress", (event) => {
+	const percent = `${((event.bytesLoaded / event.file.size) * 100)}%`;
+	$("#upload-progressbar").width(percent);
+});
+
+instance.addEventListener("error", (event) => {
+	// Reset progressbar
+	$("#upload-progressbar").width(0);
+	$("#connection-error").addClass("shown").text(event.message);
+});
+
+const $form = $(document);
+const $overlay = $("#upload-overlay");
+
+$form.on("drag dragstart dragend dragleave", () => false);
```

Comment:
Remove `dragleave` here.

And add `return false` in the next 2 events, I guess that's why you had them multiplied.

Reply:
Done